tab_automl/automl/datasets.py

```diff
@@ -53,7 +66,19 @@ class RegressionDataset:
 
     def __init__(self, path):
         # reading the dataset from source
-        self.data = pd.read_csv(path)
+        if path[-4:] == ".txt":
```

Comment:
Can you change this `[-some_number]` with `path.endswith` module?
Will be a better way to understand.